### PR TITLE
feature: contracts-ops package

### PR DESCRIPTION
**Pull request by prestwich**
## Motivation

Create a home for foundry-based maintenance scripts

## Solution

A contracts-ops package 🎉 

## PR Checklist

- [ ] Added Tests
- [x] Updated Documentation
- [x] Updated CHANGELOG.md for the appropriate package

### CODE REVIEW

File: foundry.toml [hunk 1]
```diff
@@ -70,4 +70,8 @@ out = 'packages/contracts-da-bridge/foundry_artifacts'
 ffi = true
 fuzz-runs = 10_000
 
-
+[profile.ops]
+src = 'packages/contracts-ops/contracts'
+test = 'packages/contracts-ops/contracts/test'
+out = 'packages/contracts-ops/foundry_artifacts'
+ffi = true
```
Comment (odyslam) [inline foundry.toml:77]: - Do we need ffi? 
- I think we need file permissions

File: packages/contracts-ops/package.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "author": "James Prestwich",
+  "scripts": {
+    "build": "yarn build:deps && yarn compile",
+    "build:deps": "../../scripts/install-deps.sh",
+    "clean": "FOUNDRY_PROFILE=ops forge clean",
+    "compile": "FOUNDRY_PROFILE=ops forge build",
+    "lint": "solhint contracts/*.sol",
+    "lint:fix": "solhint --fix contracts/*.sol",
+    "prettier": "prettier --write './contracts'",
+    "test:unit": "FOUNDRY_PROFILE=bridge-ci forge test",
```
Comment (odyslam) [inline packages/contracts-ops/package.json:24]: This doesn't seem right

File: packages/contracts-ops/package.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    "test:unit:local": "FOUNDRY_PROFILE=bridge forge test"
```
Comment (odyslam) [inline packages/contracts-ops/package.json:25]: Also doesn't seem right

**Review by odyslam — CHANGES_REQUESTED** [review]

File: foundry.toml (hunk 1 above)
Comment (prestwich) [reply foundry.toml:77]: gonna leave ffi in in case I want JQ

**Pull request merged**